### ISSUE TITLE
Lockfile warning during yarn install

### ISSUE THREAD

**Issue by reflog** [pikapkg/pack#3]
```
yarn install v1.13.0
[1/4] Resolving packages...
warning Lockfile has incorrect entry for "@pika/types@^0.3.3". Ignoring it.
```

**Comment by FredKSchott** [comment]
Interesting, let me test this locally and see if I can reproduce. Can you share the dependencies you added to your package.json that introduced this?

**Comment by reflog** [comment]
I only have devDeps:
```
  "devDependencies": {
    "@pika/pack": "^0.3.1",
    "@pika/plugin-build-node": "0.3.9",
    "@pika/plugin-build-types": "0.3.9",
    "@pika/plugin-build-web": "0.3.9",
    "@pika/plugin-ts-standard-pkg": "0.3.9",
    "tslint": "^5.11.0",
    "typescript": "3.3.1"
  }
```

**Comment by FredKSchott** [comment]
Looks like this was inadvertently solved by a83b7b58376ed26d624cc54d5776d681fe8bdc0e
